isisd/isis_main.c

```diff
@@ -131,9 +111,7 @@ terminate (int i)
 void
 sighup (void)
 {
-  zlog_debug ("SIGHUP received");
-  reload ();
-
+  zlog_err ("SIGHUP/reload is not implemented for isisd");
```

Comment:
hm. bgpd, ripd and ripngd actually to to reload config and restart. The others have a message "SIGHUP received" (as "info" severity).

... I'd keep the isisd one as "error" severity since it's a change compared to previous behaviour, not sure if we should expand the others...